weld/ejb/src/main/java/org/jboss/as/weld/ejb/StatefulSessionObjectReferenceImpl.java

```diff
@@ -77,7 +79,8 @@ public StatefulSessionObjectReferenceImpl(EjbDescriptorImpl<?> descriptor) {
     @SuppressWarnings({ "unchecked" })
     public synchronized <S> S getBusinessObject(Class<S> businessInterfaceType) {
         if (isRemoved()) {
-            throw WeldLogger.ROOT_LOGGER.ejbHashBeenRemoved();
+            // Cannot be placed in WeldLogger
+            throw new NoSuchEJBException("EJB has been removed");
```

Comment:
It needs i18n still.

You probably just need to create a new logger class in this weld/ejb module